src/backend/TrafficCourts/Citizen.Service/Validators/FormRecognizerValidator.cs

```diff
@@ -73,12 +76,106 @@ public static void Sanitize(OcrViolationTicket violationTicket)
         {
             string count3Section = violationTicket.Fields[OcrViolationTicket.Count3Section].Value ?? "";
             if (!violationTicket.Fields[OcrViolationTicket.Count3ActRegs].IsPopulated()
-                && count3Section.StartsWith("MVA"))
+                && count3Section.StartsWith("M") && count3Section.Length >= 3)
             {
-                violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value = "MVA";
-                violationTicket.Fields[OcrViolationTicket.Count3Section].Value = count3Section.Replace("MVA", "");
+                string actReg = count3Section.Substring(1, 3);
+                violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value = actReg;
+                violationTicket.Fields[OcrViolationTicket.Count1Section].Value = count3Section.Replace((actReg is not null ? actReg : ""), "")?.Trim();
             }
         }
+
+        // If any act regs starts with MU replace with MV since the U is likely a mis-scan of the V
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.Count1ActRegs))

```

Comment:
A lot of duplicate code. It is also long and verbose. Hard to follow. This code could be simplifeid like:

```cs
        void ReplaceMUWithMV(string fieldName)
        {
            if (violationTicket.Fields.TryGetValue(fieldName, out var field))
            {
                field.Value = field.Value?.Replace("MU", "MV");
            }
        }

        ReplaceMUWithMV(OcrViolationTicket.Count1ActRegs);
        ReplaceMUWithMV(OcrViolationTicket.Count2ActRegs);
        ReplaceMUWithMV(OcrViolationTicket.Count3ActRegs);
    ```

    Note, it is perfectly valid to have a function declared inside a method.

Reply:
thanks for your feedback -- updated to be a bit more readable hopefully